amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -91,88 +92,72 @@ class EditableText extends React.Component<EditableTextProps, EditableTextState>
   };
 
   refreshText = () => {
-    this.setState({value: this.state.refreshValue, isDisabled: false, inEditMode: false, refreshValue: undefined });
+    this.setState({ value: this.state.refreshValue, isDisabled: false, inEditMode: true, refreshValue: undefined });
   };
 
   updateText = () => {
     const newValue = this.textAreaRef.current.value;
-    const onSuccessCallback = () => { this.setState({value: newValue, inEditMode: false, refreshValue: undefined }); };
+    const onSuccessCallback = () => { this.setState({ value: newValue, inEditMode: false, refreshValue: undefined }); };
     const onFailureCallback = () => { this.exitEditMode(); };
 
     this.props.onSubmitValue(newValue, onSuccessCallback, onFailureCallback);
```

Comment:
I'm not really sure what's the best way to test the onSuccessCallback/onFailureCallback methods.